appeals/api/src/server/repositories/appeal.repository.js

```diff
@@ -86,7 +86,12 @@ const getUserAppeals = (userId, pageNumber, pageSize, status) => {
 		OR: [
 			{ inspector: { azureAdUserId: { equals: userId } } },
 			{ caseOfficer: { azureAdUserId: { equals: userId } } }
-		]
+		],
+		AND: {
```

Comment:
This query seems to work as intended (the idea is to exclude cases with appealStatus of "complete" from the returned data), but I arrived at it mostly by trial and error (not having a great understanding of prisma queries). So I'd appreciate any feedback on whether the way I've done it is correct or could be improved!

Reply:
that looks correct! as long as the OR / AND logic does what you need is the way to go...